tests/NumericalSchemaValidatorTest.php

```diff
@@ -18,17 +18,17 @@ final class NumericalSchemaValidatorTest extends BaseCodegenTestCase {
   public function testInteger(): void {
```

Comment:
This file was completely unclear if it cares about array types, defaulting to darrays.